Machine Learning/ProblemChild/datafeeds/datafeed-experimental-high-sum-by-user-problemchild.txt

```diff
@@ -0,0 +1,54 @@
+PUT _ml/datafeeds/experimental-high-sum-by-user-problemchild
```

Comment:
Remove `PUT` statements from `datafeed` files.

change file extension from `txt` to `json`